application/src/main/java/org/togetherjava/tjbot/commands/help/HelpSystemHelper.java

```diff
@@ -333,6 +333,59 @@ private void executeUncategorizedAdviceCheck(long threadChannelId, long authorId
         }).queue();
     }
 
+    void scheduleNoActivityAdviceCheck(long threadChannelId, long authorId) {
+        SERVICE.schedule(() -> {
+            try {
+                executeNoActivityAdviceCheck(threadChannelId, authorId);
+            } catch (Exception e) {
+                logger.warn(
+                        "Unknown error during a no activity advice check on thread {} by author {}.",
+                        threadChannelId, authorId, e);
+            }
+        }, SEND_NO_ACTIVITY_ADVICE_AFTER_MINUTES, TimeUnit.MINUTES);
+    }
+
+    private void executeNoActivityAdviceCheck(long threadChannelId, long authorId) {
+        logger.debug("Executing no activity advice check for thread {} by author {}.",
+                threadChannelId, authorId);
+
+        ThreadChannel threadChannel = jda.getThreadChannelById(threadChannelId);
+        if (threadChannel == null) {
+            logger.debug(
+                    "Channel for no activity advice check seems to be deleted (thread {} by author {}).",
+                    threadChannelId, authorId);
+            return;
+        }
+
+        if (threadChannel.isArchived()) {
+            logger.debug(
+                    "Channel for no activity advice check is archived already (thread {} by author {}).",
+                    threadChannelId, authorId);
+            return;
+        }
+        if (hasNoAuthorActivity(authorId, threadChannel)) {
+            MessageEmbed embed = HelpSystemHelper.embedWith(
+                    """
+                            Hey there %s👋 It has been a bit after you created this thread and you still did not share any details of your question.
+                            Helpers have seen your question already and are just waiting for you to elaborate on your problem and provide detailed information on it 👌
+                            """
+                        .formatted(User.fromId(authorId).getAsMention()));
+            threadChannel.sendMessageEmbeds(embed).queue();
+        }
+    }
+
+    private static boolean hasNoAuthorActivity(long authorId, ThreadChannel threadChannel) {
+        int count = 0;
+        for (Message message : threadChannel.getIterableHistory()) {
+            if (count == 10)
+                return false;
+            if (message.getAuthor().getIdLong() == authorId)
+                return false;
+            count++;
+        }
```

Comment:
Does this work? Have you checked or any messages actually reach this loop?

Reply:
Yes, I posted a screenshot

Reply:
> Have you checked or any messages actually reach this loop?

Screenshot only shows that the method runs, not that any messages actually reach the loop

Reply:
So you think I should log it in that case?

Reply:
> Have you checked or any messages actually reach this loop?

So I'm not asking for you to log, I'm only asking you to check it works **right now**
If it works, you can stop logging, but I just need to know or it works

Reply:
yes